### PR TITLE
Update orders API

### PR DESCRIPTION
**Pull request by andrioli**
Added new Order property `next_action` and new parameters (`total_min`, `total_max`, `customer_ids`, `q`) for endpoint GET /orders

@maqui01 Can you review?

### CODE REVIEW

File: resources/order.md [hunk 1]
```diff
@@ -6,65 +6,65 @@ An order is created when a customer completes the checkout process. Orders can't
...
+| weight                     | Order's total weight, in kilograms                                                                                                                                          |
+| status                     | Order's status. Possible values are "open", "closed" or "cancelled"                                                                                                         |
+| payment_status             | Order's payment status. Possible values are "authorized", "pending", "paid", "abandoned", "refunded" or "voided"                                                            |
+| shipping_status            | Order's shipping status. Possible values are "shipped" or "unshipped"                                                                                                       |
+| next_action                | Next available operation in the orders flow                                                                                                                                 |
```
Comment (maqui01) [inline resources/order.md]: Could you add the list of possible values of [next_action]?

File: resources/order.md [hunk 2]
```diff
@@ -75,19 +75,23 @@ Endpoints
...
+| updated_at_min | Show Orders last updated after date ([ISO 8601 format](http://en.wikipedia.org/wiki/ISO_8601))              |
+| updated_at_max | Show Orders last updated before date ([ISO 8601 format](http://en.wikipedia.org/wiki/ISO_8601))             |
+| total_min      | Show Orders with total value bigger or equals than the specified value                                      |
+| total_max      | Show Orders with total value lower or equals than the specified value                                       |
+| customer_ids   | Restrict results to the specified customer IDs (comma-separated)                                            |
+| page           | Page to show                                                                                                |
+| per_page       | Amount of results                                                                                           |
+| fields         | Comma-separated list of fields to include in the response                                                   |
+| q              | Search Orders by the givan number; or containing the given text in the customer name or email               |
```
Comment (maqui01) [inline resources/order.md]: Is this comparing only as a StartWith like products?

Comment (andrioli) [inline resources/order.md]: Nop. Is 'like %q%'

Comment (lombo) [inline resources/order.md]: Typo on first "given"

File: resources/order.md (hunk 1 above)
Comment (andrioli) [inline resources/order.md]: Soved!

**Comment by maqui01** [comment]
:+1:

**Pull request merged**